detailed_docs/iAP_over_BT/iAP_over_BT_re-connection_use_case.md

```diff
@@ -0,0 +1,88 @@
+## Use Case 1: iAP over BT re-connection
+
+**Main Flow:**  
+
+_Pre-conditions:_  
+a. iOS device is connected over Bluetooth  
+b. Application from iOS device is registered and running on SDL 
+
+_Steps:_    
+1. User connects the same iOS device over USB  
+
+_Expected:_  
+
+2. SDL receives signal with `<UUID>` about USB connection of the same device 
+3. SDL compares `<UUID>` of BT device connection with `<UUID>` of USB connection  
```

Comment:
Isn't the HMI responsible for doing the UUID check and informing SDL to perform transport switch? Please make sure to correct this entire flow accordingly

Reply:
The flow was corrected accordingly